### PR TITLE
K8SPG-294 PG operator 1.4.0 release

### PR DESCRIPTION
**Pull request by cap1984**

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

File: sources/operator.1.4.0.pg-operator.json [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+          "14.7": {
+            "image_path": "percona/percona-postgresql-operator:1.4.0-ppg14-postgres-ha",
+            "image_hash": "93b69b07914f1ed43812013b7bfc75692369bc39fc24349853584165fc192e45",
+            "status": "recommended",
+            "critical": false
+          }
+        },
+        "pmm": {
+          "2.29.0": {
+            "image_path": "percona/pmm-client:2.36.0",
```
Comment (nmarukovich) [inline sources/operator.1.4.0.pg-operator.json:29]: should be 2.36, not 2.29

Comment (cap1984) [reply sources/operator.1.4.0.pg-operator.json:29]: done

**Review by nmarukovich — APPROVED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

**Review by egegunes — APPROVED** [review]

**Review by tplavcic — APPROVED** [review]

**Pull request merged**